app/src/main/java/com/keylesspalace/tusky/components/preference/PreferencesFragment.kt

```diff
@@ -145,6 +145,13 @@ class PreferencesFragment : PreferenceFragmentCompat(), Injectable {
                     isSingleLineTitle = false
                 }
 
+                switchPreference {
```

Comment:
Something I do not understand:
* Changing this preference has no effect on the main view unless MainActivity.onCreate is re-called
* Changing this preference does not recreate MainActivity
* But changing the App Theme preference above (line 55) does recreate MainActivity (so any change to this preference takes effect)
* How do I make this preference recreate MainActivity when it changes?

Reply:
Fixed